### DISCUSSION TITLE
Future of Wireguard-Easy?

### DISCUSSION THREAD

**Discussion by altonmazin** [wg-easy/wg-easy#1215] [General]
I see a lot of potential to this interface, but a lot of lacking features, password protected GUI more robust configurations, user management etc...

I think this could be a nice product and i think if we have enough developers here that are willing to work, maybe we can get some sort of donation website spun up to make this a better product.

i've been looking around and not much exist out there for Wireguard

anyone has any suggestions or throughts?

**Comment by offizium-berndstorath** [comment]
The Gui is Password protected.

Better UI (Configuration through Web Interface) is already planned. Because of that a proper database solution will be added where user management is already considered.

Just stay tuned
